### ISSUE TITLE
Rename VK_TRUE and VK_FALSE

### ISSUE THREAD

**Issue by Cyres** [MaikKlein/ash#75]
currently, `Bool32` contains `VK_TRUE` and `VK_FALSE`. Since we will have a breaking change anyway with #52, you may consider renaming them to `TRUE` and `FALSE` just like all the other enum-structs?

**Comment by MaikKlein** [comment]
Yeah I think that makes sense and this should probably apply to other constants as well.

**Comment by Cyres** [comment]
There are more constants, which should be renamed: 

- `VK_MAX_PHYSICAL_DEVICE_NAME_SIZE`
- `VK_UUID_SIZE`
- `VK_LUID_SIZE`
- `VK_MAX_EXTENSION_NAME_SIZE`
- `VK_MAX_DESCRIPTION_SIZE`
- `VK_MAX_MEMORY_TYPES`
- `VK_MAX_MEMORY_HEAPS`
- `VK_LOD_CLAMP_NONE`
- `VK_REMAINING_MIP_LEVELS`
- `VK_REMAINING_ARRAY_LAYERS`
- `VK_WHOLE_SIZE`
- `VK_ATTACHMENT_UNUSED`
- `VK_QUEUE_FAMILY_IGNORED`
- `VK_QUEUE_FAMILY_EXTERNAL`
